frontend/packages/topology/package.json

```diff
@@ -21,6 +21,12 @@
       "topology": [
         "integration-tests/**/*.scenario.ts"
       ]
+    },
+    "exposedModules": {
+      "reduxReducer": "src/utils/reducer.ts"
+    },
+    "dependencies": {
```

Comment:
No need to specify `consolePlugin.dependencies` meta-object for static plugins.

Just `consolePlugin.exposedModules` should be enough to process the static plugin's `console-extensions.json` files. Static plugins don't have a dynamic plugin manifest JSON which is where these dependencies are reflected.